tests/e2e/internal/retrieval/test_series.py

```diff
@@ -1,61 +1,117 @@
+    return ensure_project(
+        client=client,
+        api_token=api_token,
+        workspace=workspace,
+        unique_key=test_execution_id,
+        project_data=ProjectData(
+            project_name_base="project_404",
+            runs=[
+                RunData(
+                    run_id_base="run_project_1_alpha",
+                    fork_point=None,
+                    string_series={
+                        "metrics/strfoobar_1": {i: f"string-1-{i}" for i in range(10)},
+                        "metrics/strfoobar_2": {i: f"string-2-{i}" for i in range(10)},
+                    },
+                    histogram_series={
+                        "metrics/histograms_1": {
+                            0: Histogram(bin_edges=[1, 2, 3, 4], counts=[10, 20, 30]),
+                            1: Histogram(bin_edges=[4, 5, 6, 7], counts=[40, 50, 60]),
+                            2: Histogram(bin_edges=[7, 8, 9, 10], counts=[70, 80, 90]),
+                            3: Histogram(bin_edges=[11, 12, 13, 14], counts=[10, 20, 30]),
+                            4: Histogram(bin_edges=[14, 15, 16, 17], counts=[40, 50, 60]),
+                            5: Histogram(bin_edges=[17, 18, 19, 20], counts=[70, 80, 90]),
+                        },
+                    },
+                    file_series={
+                        "file-series/file_series_1": {
+                            0: File(b"file-1-0", mime_type="text/plain"),
+                            1: File(b"file-1-1", mime_type="text/plain"),
+                            2: File(b"file-1-2", mime_type="text/plain"),
+                        },
+                        "file-series/file_series_2": {
+                            0: File(b"file-2-0", mime_type="text/plain"),
+                            1: File(b"file-2-1", mime_type="text/plain"),
+                            2: File(b"file-2-2", mime_type="text/plain"),
+                        },
+                        "file-series/file_series_3": {
+                            0: File(b"file-3-0", mime_type="text/plain"),
+                            1: File(b"file-3-1", mime_type="text/plain"),
+                            2: File(b"file-3-2", mime_type="text/plain"),
+                        },
+                    },
+                ),
+            ],
         ),
-        (FILE_SERIES_PATHS[0], "file_series", TEST_DATA.experiments[0].file_series_matchers()[FILE_SERIES_PATHS[0]]),
-    ],
-)
-def test_fetch_series_values_single_series(
-    client, project, experiment_identifier, attribute_name, attribute_type, expected_values
-):
+    )
+
+
+def test_fetch_series_values_does_not_exist(client, project_1):
     # given
-    run_definition = RunAttributeDefinition(experiment_identifier, AttributeDefinition(attribute_name, attribute_type))
+    run_definition = RunAttributeDefinition(
+        RunIdentifier(project_1.project_identifier, project_1.ingested_runs[0].experiment_name),
```

Comment:
Pls wrap project identifier in `ProjectIdentifier()`.

Also, `experiment_name` is not sys_id - both type and value are incorrect.